engine/src/main/java/org/hibernate/search/engine/search/aggregation/spi/AggregationTypeKeys.java

```diff
@@ -18,5 +18,17 @@ private AggregationTypeKeys() {
 			of( IndexFieldTraits.Aggregations.TERMS );
 	public static final SearchQueryElementTypeKey<RangeAggregationBuilder.TypeSelector> RANGE =
 			of( IndexFieldTraits.Aggregations.RANGE );
+	public static final SearchQueryElementTypeKey<MetricFieldAggregationBuilder.TypeSelector> SUM =
+			of( IndexFieldTraits.Aggregations.SUM );
+	public static final SearchQueryElementTypeKey<MetricFieldAggregationBuilder.TypeSelector> MIN =
+			of( IndexFieldTraits.Aggregations.MIN );
+	public static final SearchQueryElementTypeKey<MetricFieldAggregationBuilder.TypeSelector> MAX =
+			of( IndexFieldTraits.Aggregations.MAX );
+	public static final SearchQueryElementTypeKey<SearchAggregationFilteredBuilder<Long>> COUNT =
+			of( IndexFieldTraits.Aggregations.COUNT );
+	public static final SearchQueryElementTypeKey<SearchAggregationFilteredBuilder<Long>> COUNT_DISTINCT =
+			of( IndexFieldTraits.Aggregations.COUNT_DISTINCT );
+	public static final SearchQueryElementTypeKey<SearchAggregationFilteredBuilder<Double>> AVG =
+			of( IndexFieldTraits.Aggregations.AVG );
```

Comment:
I think the names you've selected are inconsistent with existing naming conventions.

* `MetricFieldAggregationBuilder` => `FieldMetricAggregationBuilder`
* `SearchAggregationFilteredBuilder` => `SearchFilteredAggregationBuilder` ... And I'd even say: `SearchFilterableAggregationBuilder`, since it's not necessarily filtered.